Cliqz/Upgrade&Payment/View/UpgradeViews/BaseUpgradeViewController.swift

```diff
@@ -126,14 +141,44 @@ class BaseUpgradeViewController: UIViewController {
         }
     }
 
-    private func startLoadingAnimation() {
-        self.loadingView.startAnimating()
+    // MARK: - Loading
+    func startLoadingAnimation() {
+
+        loadingOverlay.alpha = 0
+        view.addSubview(loadingOverlay)
```

Comment:
maybe loadingOverlay.removeFromSuperView() before adding? i see, the stop animation is removing and theoretically there is no need, but start will be called my mistake twice in raw without stop, then it can lead to a crash